pandas/tests/extension/json/test_json.py

```diff
@@ -100,7 +100,7 @@ def assert_frame_equal(self, left, right, *args, **kwargs):
             check_names=kwargs.get("check_names", True),
             check_exact=kwargs.get("check_exact", False),
             check_categorical=kwargs.get("check_categorical", True),
-            obj="{obj}.columns".format(obj=kwargs.get("obj", "DataFrame")),
+            obj=f"{kwargs.get('obj', 'DataFrame')}.columns",
```

Comment:
can you take the `kwargs.get` out to a separate line

Reply:
Since keyword arguments are passed to `tm.assert_index_equal`, I didn't really know where best to put the new line for the `kwargs.get` call, so for the time being I put it before the call to `tm.assert_index_equal`. Also I'm not too sure if `obj_type` is a good binding name for this value. Please review this change.